docs/searchpredicates.txt

```diff
@@ -110,6 +110,14 @@ Geoshape.circle(37.97, 23.72, 50)
 Geoshape.box(37.97, 23.72, 38.97, 24.72)
 //WKT (requires JTS for polygon, linestring and multi-point/line/polygon geometries)
 Geoshape.fromWkt("POLYGON ((35.4 48.9, 35.6 48.9, 35.6 49.1, 35.4 49.1, 35.4 48.9))")
+//MultiPoint
+Geoshape.geoshape(Geoshape.HELPER.getContext().getShapeFactory().multiPoint().pointXY(60.0, 60.0).pointXY(120.0, 60.0).build());
```

Comment:
Would it make sense to expose helper Geoshape methods like `multiPointBuilder()` so this (and similarly below) would be `Geoshape.multiPointBuilder().pointXY(60.0, 60.0).pointXY(120.0, 60.0).build()`? Or if not how about `Geoshape.getShapeFactory()`?